packages/react-core/src/components/AboutModal/AboutModal.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
+import AboutModalBoxBackgroundImage from '@patternfly/react-tokens/dist/esm/c_about_modal_box_BackgroundImage';
```

Comment:
```suggestion
import backgroundImage from '@patternfly/react-tokens/dist/esm/c_about_modal_box_BackgroundImage';
```

Just looking at the format of other token imports, we can probably write this shorter and with a lowercase first letter

```
Pagination.tsx:import widthChars from '@patternfly/react-tokens/dist/esm/c_pagination__nav_page_select_c_form_control_width_chars';
Page.tsx:import globalBreakpointXl from '@patternfly/react-tokens/dist/esm/global_breakpoint_xl';
Tooltip.tsx:import tooltipMaxWidth from '@patternfly/react-tokens/dist/esm/c_tooltip_MaxWidth';
Alert.tsx:import maxLines from '@patternfly/react-tokens/dist/esm/c_alert__title_max_lines';
ToolbarToggleGroup.tsx:import globalBreakpointLg from '@patternfly/react-tokens/dist/esm/global_breakpoint_lg';
ToolbarUtils.tsx:import globalBreakpointMd from '@patternfly/react-tokens/dist/esm/global_breakpoint_md';
ToolbarUtils.tsx:import globalBreakpointLg from '@patternfly/react-tokens/dist/esm/global_breakpoint_lg';
ToolbarUtils.tsx:import globalBreakpointXl from '@patternfly/react-tokens/dist/esm/global_breakpoint_xl';
ToolbarUtils.tsx:import globalBreakpoint2xl from '@patternfly/react-tokens/dist/esm/global_breakpoint_2xl';
Popover.tsx:import popoverMaxWidth from '@patternfly/react-tokens/dist/esm/c_popover_MaxWidth';
Popover.tsx:import popoverMinWidth from '@patternfly/react-tokens/dist/esm/c_popover_MinWidth';
MenuItem.tsx:import topOffset from '@patternfly/react-tokens/dist/esm/c_menu_m_flyout__menu_top_offset';
MenuItem.tsx:import rightOffset from '@patternfly/react-tokens/dist/esm/c_menu_m_flyout__menu_m_left_right_offset';
MenuItem.tsx:import leftOffset from '@patternfly/react-tokens/dist/esm/c_menu_m_flyout__menu_left_offset';
NotificationDrawerGroup.tsx:import maxLines from '@patternfly/react-tokens/dist/esm/c_notification_drawer__group_toggle_title_max_lines';
NotificationDrawerListItemHeader.tsx:import maxLines from '@patternfly/react-tokens/dist/esm/c_notification_drawer__list_item_header_title_max_lines';
ModalBox.tsx:import topSpacer from '@patternfly/react-tokens/dist/esm/c_modal_box_m_align_top_spacer';
ExpandableSection.tsx:import lineClamp from '@patternfly/react-tokens/dist/esm/c_expandable_section_m_truncate__content_LineClamp';
JumpLinks.tsx:import cssToggleDisplayVar from '@patternfly/react-tokens/dist/esm/c_jump_links__toggle_Display';
```